moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/parameterization/joint_space/constrained_planning_state_space.h

```diff
@@ -0,0 +1,76 @@
+/*********************************************************************
```

Comment:
I put it in the joint_space folder because it is a "joint space" state space, not a "constrained space". But I do agree the name is confusing. I can't find a better name to describe "This is the joint model state space you should use to created a constrained state space from, but this state space itself is *not* constrained." Any ideas?

Now that you mention it, it seems to mee that maybe the folder structure is a bit overkill. We could have all the files in the folder `parameterization`. Especially because I will possibly remove the state space factories, so there will be only 3 files left.